### PR TITLE
Use npm 3 in travis to solve tests for #269

### PR DESCRIPTION
**Pull request by josemarluedke**
This fixes the tests on PR #269.

cc @rwjblue @danmcclain.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ cache:
 
 before_install:
   - npm config set spin false
-  - npm install -g npm@^2
+  - npm install -g npm@^3
```
Comment (kellyselden) [inline .travis.yml]: I think you can remove this line to get npm 3 automatically.

**Comment by kratiahuja** [comment]
Does this mean we are only going to support npm 3? Just want to make sure we call it out explicitly.

**Comment by josemarluedke** [comment]
@kratiahuja Don't think ember-cli-fastboot is only compatible with NPM 2, just that the test suite breaks when using NPM 2. Right now, the test suite is broken for users that are using NPM 3 or above.

File: .travis.yml (hunk 1 above)
Comment (josemarluedke) [inline .travis.yml]: Will update!

**Pull request merged**